beam_nuggets/io/relational_db.py

```diff
@@ -22,7 +22,7 @@
 assert TableConfiguration is not None
 
 
-class Read(PTransform):
+class DBRead(PTransform):
```

Comment:
If not mistaken, this is related to [this issue](https://github.com/mohaseeb/beam-nuggets/issues/19)? 
If so, [I couldn't actually reproduce the issue](https://github.com/mohaseeb/beam-nuggets/issues/19#issuecomment-504703433) using a recent apache-beam version `2.13.0`. What beam version are you using?

Reply:
Changing Read to DBRead was my solution to the issue you linked, there might be a better way around that error but I have not found it. I am using apache-beam 2.13.0 and I only come across this issue when deploying to Dataflow. I am not using `with beam.Pipeline(options=options) as p:`. 

The error being thrown in [this issue](https://github.com/mohaseeb/beam-nuggets/issues/19) appears to be Dataflow thinking that the Read PTransform is actually [this PTransform](https://beam.apache.org/releases/pydoc/2.13.0/_modules/apache_beam/io/iobase.html#Read) instead. I have experimented with importing the beam_nuggets Read PTransform both locally and globally in my pipeline with the same results. 

When I am deploying a pipeline to Dataflow I am also specifying a `setup.py` file instead of a `requirements.txt`. I don't know if that would change the outcome of your test.

Reply:
I totally missed the part this issue is related to Read transform not Write. 
I think your analysis and solution make sense; I'll merge it. I suggest though we call the transform something like `ReadFromDB` following the pattern of built it ones like `ReadFromText` and `ReadFromDatastore`.